djautotask/api_rest.py

```diff
@@ -451,3 +451,17 @@ def update_task(self, task, changed_fields):
 
     def _endpoint(self):
         return '{}{}'.format(self.api_base_url, self.POST_QUERY)
+
+
+class ProjectsAPIClient(AutotaskAPIClient):
+    API = 'Projects'
+
+    def get_project(self, project_id):
+        return self.get_instance(project_id)
+
+    # use POST method because of IN-clause query string
+    def get_projects(self, next_url, *args, **kwargs):
+        return self.fetch_resource(next_url, method='post', *args, **kwargs)
```

Comment:
I'm seeing the `endpoint_url` as `https://webservices2.autotask.net/ATServicesRest/v1.0/Projects/query?search={"filter": [{"op": "noteq", "field": "status", "value": 5}, {"op": "in", "field": "status", "value": [0, 1, 2, 3, 4, 5, 6, 7]}]}`. 


I'd expect to see it as ``https://webservices2.autotask.net/ATServicesRest/v1.0/Projects/query` since it's a POST request.

Reply:
I think you need to add an `_endpoint` method to `ProjectsAPIClient`

Reply:
Oh, I couldn't catch that. Thanks Cameron.